src/pseudopeople/noise_entities.py

```diff
@@ -16,6 +16,9 @@ class __NoiseTypes(NamedTuple):
     """
 
     omission: RowNoiseType = RowNoiseType("omit_row", noise_functions.omit_rows)
+    do_not_respond: RowNoiseType = RowNoiseType(
+        "do_not_respond", noise_functions.omit_target_rows
```

Comment:
We talked before about making both the `omission` and `do_not_respond` noise types have the same noise function and using the dataset name which is an argument to the noise function to determine whether or not to use targeted omission. I still think that's better, since it makes it more explicit that targeted omission is just the omission noise function for census, acs, and cps.

Reply:
This was decided by RT to some extent. They wanted omit_rows and do_not_respond as separate types. At that point it was natural to maintain two separate functions for two separate noise types.

Reply:
Let's talk about this tomorrow when we're talking about the tests.

Reply:
Memorializing: We resolved not to use the layer of indirection of a shared `omit_rows` `do_not_respond` function and instead check that the required columns for the `do_not_respond` method are checked for existence in the dataset.